mani_skill/examples/motionplanning/base_motionplanner/motionplanner.py

```diff
@@ -0,0 +1,243 @@
+import mplib
+import numpy as np
+import sapien
+import trimesh
+
+from mani_skill.agents.base_agent import BaseAgent
+from mani_skill.envs.sapien_env import BaseEnv
+from mani_skill.envs.scene import ManiSkillScene
+from mani_skill.utils.structs.pose import to_sapien_pose
+import sapien.physx as physx
+from transforms3d import quaternions
+
+
+class BaseMotionPlanningSolver:
+    OPEN = 1
```

Comment:
these fixed constants are very specific two finger grippers. Hence we should rename `BaseMotionPlanningSolver` to something about two finger gripper robots.

A better refactor is to still keep this motion planning solver, but abstract the grasp related information out (for robots like the stick panda robot). Then make one for two finger robots specifically.